.github/workflows/build.yml

```diff
@@ -8,20 +8,28 @@ on:
   pull_request:
 
 jobs:
+
   linux:
+
     runs-on: ubuntu-22.04
+
     strategy:
       matrix:
         config:
         # if there are multiple items in this list, only use should
         # deployit=true for just one of them.
-         - {grb_version: 9.3.1, deployit: true}
+         - {grb_version: 9.0.0, deployit: false}
+         - {grb_version: 10.0.3, deployit: true}
```

Comment:
Before we release a stable v1.2, I will have a GraphBLAS v10.1.0 ready. That version of GraphBLAS has an important feature: the ability to force GraphBLAS.h to not expose any GxB methods to the user application, with

#define GRAPHBLAS_VANILLA
#include <GraphBLAS.h>

That addition to GraphBLAS enabled me to find some GxB's that had snuck into the LAGraph "vanilla" build.